src/sagemaker/local/utils.py

```diff
@@ -1,4 +1,4 @@
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please remove the dates

Reply:
Sorry about that, not sure what happened there. Fork was probably out of sync.